metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_listener.py

```diff
@@ -687,9 +688,15 @@ def on_dag_run_running(self, dag_run: "DagRun", msg: str) -> None:
                 f"DataHub listener got notification about dag run start for {dag_run.dag_id}"
             )
 
-            self.on_dag_start(dag_run)
```

Comment:
this logic should be applied to the task run events as well. we can check the dag that the task is part of